client/src/components/input/index.js

```diff
@@ -13,14 +14,26 @@ class Input extends Component {
   }
 
   render() {
-    const { labelText, placeholder } = this.props;
+    const { labelText, placeholder, type } = this.props;
     return (
       <div className='container-input'>
         <label className='label-style'>{labelText}</label>
-        <input className='input-style' placeholder={placeholder} onChange={this.inputValue} />
+        <input
+          className='input-style'
+          placeholder={placeholder}
+          type={type}
+          onChange={this.inputValue}
```

Comment:
I think this.props.onTextInputChange is the change handler you want.  I think this.inputValue is probably undefined - I don't see it being set anywhere.  But you probably want to set the input element's value to {this.props.value}, to let consumers of this input component pass values in.